frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -384,6 +422,34 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
         sessionPlayerSnapshotData: [
             null as SessionPlayerSnapshotData | null,
             {
+                pollRecordingSnapshots: async (_, breakpoint: BreakPointFunction) => {
+                    const params = toParams({
+                        source: 'realtime',
```

Comment:
L341 also uses this value, let's move to a const?

Reply:
hmmm, I'm not a massive fan of all repeated values being const (I think there's tension between repetition of the value and duplication between the const declaration and usage e.g. `const REALTIME = 'realtime')

that said I can be clever with TypeScript types here so we can safely reference valid source types

Reply:
I am more of the opposite because of https://en.wikipedia.org/wiki/Defensive_programming
It's very easy to mistype a word and introduce bugs.
For magical numbers, more because of eg `time: 100`, is 100 since epoch, is seconds, is milliseconds? so `const time_millis = 100` usually tells me right away, also because of defensive programming.

Reply:
![pirate-code](https://github.com/PostHog/posthog/assets/984817/8d3f0c0a-d63b-40d1-b264-8e19b8ccaa7f)